boards/beagle/beagleplay_cc1352/support/openocd.cfg

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# TI CC13x2 LaunchPad Evaluation Kit
+#
+# Since bcf does not have a built in debugger, we use XDS110 in launchpad
```

Comment:
I suggest using "BeagleConnect Freedom", rather than "bcf". Also, "LaunchPad", rather than "launchpad".

Reply:
Er, I even forgot this isn't even "BeagleConnect Freedom", it is "BeaglePlay". I suggest "BeaglePlay CC1352" here.